server/container_create.go

```diff
@@ -38,6 +38,7 @@ import (
 const (
 	seccompUnconfined      = "unconfined"
 	seccompRuntimeDefault  = "runtime/default"
+	seccompDockerDefault   = "docker/default"
```

Comment:
We will not be shipping the docker/default, at least not upstream Docker default.  Should we change the name?  Is it ok for us to add the 3 or 4 syscalls that we believe should be in the default to run systemd in a container?
Or is the RuntimeDefault our default?

Reply:
We can support the annotation using our own, I don't see why we need to break just because kube introduces docker'isms - see here https://github.com/kubernetes/kubernetes/pull/49178#issuecomment-328601346